force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +242,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
-
+        List<integer> newIntList = new List<integer>();
         // Add the integers 1, 2, and 3 to the list
-        // add 1
+        newIntList.add(1);
+        newIntList.add(2);
+        newIntList.add(3);
+        // add 1 
         // add 2
         // add 3
-
+        
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        return newIntList; // Replace null with the variable you used to store the result
```

Comment:
good. You can be even more concise and just use the List constructor. 

  return new List<Integer>{1, 2, 3};